pkg/orm/orm.go

```diff
@@ -23,19 +22,17 @@ var defaultMySQLLogger = logger.Discard.LogMode(logger.Silent) // rely on Opente
 var defaultSQLiteLogger = logger.Default.LogMode(logger.Info)
 
 // OrmConfig - configuration structure for config values at ORM module
-// TODO: Rename to Config
 type OrmConfig struct {
```

Comment:
package name is "orm" and config name could be just Config :)

Reply:
That would be a breaking change